exec: add page (#2800)
tldr-pages/tldr: pages/common/exec.md

# exec

> Replace the current process with another process.

> - Replace with specified command using the current environment variable:

Reviewer: Perhaps this will read better?
```suggestion
- Replace with the specified command using the current environment variables:
```

`exec {{command -with -flags}}`

> - Replace with specified command using the initialized environment variable:

Reviewer: Perhaps this will read better?
```suggestion
- Replace with the specified command using an empty environment:
```

`exec -c {{command -with -flags}}`

> - Replace with specified command and login to the default shell:

Reviewer: Perhaps this will read better?

```suggestion
- Replace with the specified command and login to the default shell:
```

`exec -l {{command -with -flags}}`

> - Replace with specified command and rename the process name:

Reviewer: Perhaps this will read better?
```suggestion
- Replace with the specified command and set the process name:
```
